utils/src/main/java/org/nervos/ckb/utils/Numeric.java

```diff
@@ -0,0 +1,217 @@
+package org.nervos.ckb.utils;
+
+import org.nervos.ckb.exceptions.MessageDecodingException;
+import org.nervos.ckb.exceptions.MessageEncodingException;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.util.Arrays;
+
+/**
+ * <p>Message codec functions.</p>
+ *
+ * <p>Implementation as per https://github.com/ethereum/wiki/wiki/JSON-RPC#hex-value-encoding</p>
```

Comment:
Same as above. Do we really need this at the moment?

Reply:
Have deleted